.github/workflows/identify-new-migrations.yml

```diff
@@ -0,0 +1,186 @@
+        restore-keys: ${{ runner.os }}-pip-
+
+    - name: Install Python dependencies
+      run: |
+        make dev-requirements
+        if [[ "${{ matrix.django-version }}" != "pinned" ]]; then
+          pip install "django~=${{ matrix.django-version }}.0"
+          pip check # fail if this test-reqs/Django combination is broken
+        fi
+
+    - name: Run migrations on master branch
+      env:
+        LMS_CFG: lms/envs/minimal.yml
+        # This is from the LMS dir on purpose since we don't need anything different for the CMS yet.
+        STUDIO_CFG: lms/envs/minimal.yml
+      run: |
+        echo "Running the LMS migrations."
+        ./manage.py lms migrate
+        echo "Running the CMS migrations."
+        ./manage.py cms migrate
+
+    - name: Verify executed migrations on master.
+      shell: bash
+      run: |
+        query_result1=$(mysql -h 127.0.0.1 -uedxapp001 -ppassword -e "select count(*) from django_migrations;" edxapp;)
+        echo "echo $query_result1"
+        numeric_result=$(echo "$query_result1" | tr -d '\n' | sed -E 's/[^0-9]+//g')
+        echo "Numeric Result: $numeric_result"
+        echo "FIRST_QUERY=$numeric_result" >> $GITHUB_ENV
+
+    - name: Checkout your branch repo
+      uses: actions/checkout@v2
+
+    - name: Install Python dependencies
+      run: |
+        make dev-requirements
+        if [[ "${{ matrix.django-version }}" != "pinned" ]]; then
+          pip install "django~=${{ matrix.django-version }}.0"
+          pip check # fail if this test-reqs/Django combination is broken
+        fi
+
+    - name: list installed package versions
+      run: |
+        pip freeze
+
+    - name: Run Tests
+      env:
+        LMS_CFG: lms/envs/minimal.yml
+        # This is from the LMS dir on purpose since we don't need anything different for the CMS yet.
+        STUDIO_CFG: lms/envs/minimal.yml
+      run: |
+        echo "Running the LMS migrations."
+        ./manage.py lms makemigrations
+        ./manage.py lms migrate
+        echo "Running the CMS migrations."
+        ./manage.py cms makemigrations
+        ./manage.py cms migrate
+
+    - name: Verify executed migrations on your branch.
+      shell: bash
+      id: capture2
+      run: |
+        query_result1=$(mysql -h 127.0.0.1 -uedxapp001 -ppassword -e "select count(*) from django_migrations;" edxapp;)
+        echo "echo $query_result1"
+        numeric_result=$(echo "$query_result1" | tr -d '\n' | sed -E 's/[^0-9]+//g')
+        echo "Numeric Result: $numeric_result"
+        echo "SECOND_QUERY=$numeric_result" >> $GITHUB_ENV
+
+    - name: Process Data
+      run: |
+        number2=${{ env.SECOND_QUERY }}
+        number1=${{ env.FIRST_QUERY }}
+        diff=$((number2 - number1))
+        if [ $number2 -gt $number1 ]; then
+          data=$(mysql -h 127.0.0.1 -uedxapp001 -ppassword -e "select name from django_migrations ORDER by -id limit $diff;" edxapp;)
```

Comment:
We should also select and display `app_label` so it's clear where each migration came from.